draft-ietf-taps-interface.md

```diff
@@ -577,14 +578,16 @@ digits `0-9`, the hyphen `-`, and the underscore `_`. These names serve two purp
 Transport Property Names are hierarchically organized in the
 form \[\<Namespace>.\]\<PropertyName\>.
 
-- The Namespace component MUST be empty for well-known, generic properties, i.e., for
-  properties that are not specific to a protocol.
+- The optional Namespace component and its trailing character `.` MUST be omitted for well-known,
+  generic properties, i.e., for properties that are not specific to a protocol.
 - Protocol-specific Properties MUST use the protocol acronym as the Namespace (e.g., a
-  `tcp` Connection could support a TCP-specific Transport Property, such as the user timeout
-  value, in a Protocol-specific Property called `tcp.userTimeoutValue` (see {{tcp-uto}})).
+  `TCP` Connection could support a TCP-specific Transport Property, such as the user timeout
+  value, in a Protocol-specific Property called `TCP.userTimeoutValue` (see {{tcp-uto}})).
 - Vendor or implementation specific properties MUST use a string identifying
-  the vendor or implementation as the Namespace.
-- For IETF protocols, the name of a Protocol-specific Property SHOULD be specified in an IETF document published in the RFC Series.
+  the vendor or implementation as the Namespace. This string MUST NOT start with a capital letter,
```

Comment:
We originally did not make any assumption about case to better blend into the language ecosystem. My mental model was always to treat them as case-insensitive. 

Using case as differentiator between vendor specific and standard attributes looks like a bad choice to me.

Reply:
We do need a method to differentiate allocations for IETF namespaces conflicting with vendor namespaces ... can you suggest a better one? There are surely other options.

When it comes to using a property, it is possible both are simply treated the same.

... Note: all but one of the entries in the cited IETF registry that is to be used for the IETF part of the namespace are already upper case.